### ISSUE TITLE
Bug: I want to improve `LastPlayedWidgetView` layout.

### ISSUE THREAD

**Issue by thai-d-v** [TortugaPower/BookPlayer#992]
## Environment

- OS Version: [16.6]
- App Version: [5.0.7-3]

## Description

![Back-2023-08-14-2054-3](https://github.com/TortugaPower/BookPlayer/assets/46838577/a55df47f-b106-4a87-9e19-78bf6899ad75)
- In my opinion, the title should align with the artwork to make the layout more coherent.
- I also made some changes in the `body` code to make the `ViewBuilder` simpler and more optimized.

Of course, again, I already fixed it and I'm waiting to hear from you to create the pull request. 😁

**Comment by GianniCarlo** [comment]
Hmmmm for context, the initial inspiration for this widget was from the Music app (attached screenshot with also Audible widget). I think we can improve on the existing design by aligning the text left, include the author as well on the next line, and allow only one line per field 🤔 let me know what you think @thai-d-v 

<div display="inline-block">
    <img width="350" src="https://github.com/TortugaPower/BookPlayer/assets/14112819/cec24fe6-c4d7-406f-a594-a661487ffb05">
</div>

**Comment by GianniCarlo** [comment]
yup! that's what I had in mind, although now looking at it, the author label actually limits more the title label 🤔 and that was the main thing I had against the initial proposal of having the same bounds as the artwork 😅. Not sure if @pichfl can weigh in on this, as probably I could have missed the finer details about alignment from the original designs when I implemented this

**Comment by thai-d-v** [comment]
@GianniCarlo
The reason I created this issue is that I believe there may have been an unintentional layout misalignment, and I would like to help fix it. I do not wish to propose any design changes. If the current design is what was intended, please feel free to close this issue. I am completely fine with that. 😊
